doc/devel/communication_guide.rst

```diff
@@ -0,0 +1,181 @@
+.. _communications-guidelines:
+
+========================
+Communication guidelines
+========================
+
+These guidelines are applicable when acting as a representative of Matplotlib,
+for example at sprints or when giving official talks or tutorials, and in any
+community venue managed by Matplotlib.
+
+
+.. _communication-channels:
+
+Official communication channels
+===============================
+The following venues are managed by Matplotlib maintainers and contributors:
+
+* library and docs: https://github.com/matplotlib/matplotlib
+* forum: https://discourse.matplotlib.org/
+* chat: `https://matrix.to/#/#matplotlib:matrix.org <https://matrix.to/#/#matplotlib:matrix.org>`_
+* blog: https://blog.scientific-python.org/
+
+.. _social-media:
+
+Social media
+------------
+
+Active social media
+^^^^^^^^^^^^^^^^^^^
+
+* https://twitter.com/matplotlib
+* https://instagram.com/matplotart/
+
+Official accounts
+^^^^^^^^^^^^^^^^^
+* https://bsky.app/profile/matplotlib.bsky.social
+* https://fosstodon.org/@matplotlib
+* https://www.tiktok.com/@matplotart
+* https://www.youtube.com/matplotlib
+
+
+.. _mailing-lists:
+
+Mailing lists
+----------------
+
+* `matplotlib-announce@python.org <https://mail.python.org/mailman/listinfo/matplotlib-announce>`_
+* `matplotlib-users@python.org <https://mail.python.org/mailman/listinfo/matplotlib-users>`_
+* `matplotlib-devel@python.org <https://mail.python.org/mailman/listinfo/matplotlib-devel>`_
+
+.. _social-media-coordination:
+
+Social media coordination
+-------------------------
+* team mailing list: matplotlib-social@numfocus.org
+* public chat room: `https://matrix.to/#/#matplotlib_community:gitter.im <https://matrix.to/#/#matplotlib_community:gitter.im>`_
+
+
+Maintenance
+-----------
+
+If you are interested in moderating the chat or forum or accessing the social media accounts:
+
+* matplotlib maintainer: reach out to the `community manager <https://matplotlib.org/governance/people.html#deputy-project-leads>`_ to elevate privileges.
+* everyone else: send an email explaining why to matplotlib-social-admin@numfocus.org
+
+Content guidelines
+====================
+Communication on official channels, such as the Matplotlib homepage or on
+Matplotlib social accounts, should conform to the following standards:
+
+* be primarily about Matplotlib, 3rd party packages, and visualizations made with Matplotlib
+* also acceptable topics: Python, Visualization, NumFOCUS libraries
+* no gratuitous disparaging of other visualization libraries and tools; criticism is acceptable so long as it serves a constructive purpose
+* follow visualization communication best practices:
+
+  * don't share non-expert visualizations when it could be harmful
+  * put on meeting agenda when answer isn't clearly to hold off on sharing.
```

Comment:
This is not a "visualization communication best practices"

Also, the wording is a bit unclear. Which answer? Which meeting? Do you mean "If it is unclear whether content can be published under the above rules, the topic should be discussed in the Matplotlib developer meeting"?